src/core/func-core-tools.spec.ts

```diff
@@ -66,9 +66,12 @@ describe("funcCoreTools", () => {
       expect(isCoreToolsVersionCompatible(4, 16)).toBe(true);
       expect(isCoreToolsVersionCompatible(3, 16)).toBe(false);
       expect(isCoreToolsVersionCompatible(2, 16)).toBe(false);
-      expect(isCoreToolsVersionCompatible(4, 17)).toBe(false);
+      expect(isCoreToolsVersionCompatible(4, 17)).toBe(true);
```

Comment:
Can you share a link mentioning support for Node 17?

From the docs, it's only mentioning LTS versions:
![image](https://user-images.githubusercontent.com/1699357/190584387-bbc203c5-5f7d-45b7-8d70-2c9e31af6c8a.png)

Reply:
It isn't, but I was copying the pattern above where v4 would allow node15 (line 64) and v3 would allow node13 (line 60).